packages/console/src/scss/_colors.scss

```diff
@@ -294,4 +308,5 @@
   --color-hover-danger: #feb9af; // ER40 + 8% N10
   --color-pressed_danger: #febcb2; // ER40 + 12% N10
   --color-focused-danger: rgba(255, 180, 169, 16%); // 16% ER40
+  --color-shadow-12: rgba(0, 0, 0, 12%);
```

Comment:
This might not be accurate. Please check UX design

Reply:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/5717882/163341511-032d3d33-b5b2-4cff-a4cc-fa31ddd7422c.png">